shared/js/background/load.es6.js

```diff
@@ -61,7 +61,7 @@ function loadExtensionFile (params) {
             xhr.setRequestHeader('If-None-Match', params.etag)
         }
     } else {
-        // set type xhr type tag. Safari internal xhr requests
+        // set type xhr type tag.
         // don't set a 200 status so we'll check this type
         xhr.type = 'internal'
```

Comment:
@jdorweiler I think this else statement can be dropped actually, could you confirm please?

Reply:
Actually I've checked, ignore this :)